bindings/c/test/apitester/TesterWorkload.cpp

```diff
@@ -172,7 +172,7 @@ void WorkloadBase::error(const std::string& msg) {
 	fmt::print(stderr, "[{}] ERROR: {}\n", workloadId, msg);
 	numErrors++;
 	if (numErrors > maxErrors && !failed) {
-		fmt::print(stderr, "[{}] ERROR: Stopping workload after {} errors\n", workloadId, numErrors);
+		fmt::print(stderr, "[{}] ERROR: Stopping workload after {} errors\n", workloadId, static_cast<int>(numErrors));
```

Comment:
@sfc-gh-vgasiunas, pls review the change as an author of previous change. Thx.

Reply:
Use `numErrors.load()` instead, static cast is not correct here

Reply:
Fixed. Built successful.
Thx.